ui/app/src/components/PublishDialog/utils.ts

```diff
@@ -50,7 +50,7 @@ export interface PublishDialogContainerProps
     Pick<PublishDialogProps, 'isSubmitting' | 'onSuccess' | 'onClose'> {}
 
 export interface InternalDialogState {
-  emailOnApprove: boolean;
+  packageTitle: string;
```

Comment:
Remember to keep track of **ALL THESE CHANGES** on the changelog. Don't leave for the end. Go through PR now and track them.